final/pkg/handle/handler.go

```diff
@@ -0,0 +1,74 @@
+package handle
```

Comment:
Handlers should not be part of pkg. This `pkg` holds all the utilities and third party services that the handlers will require. You can have a controllers or api directory for the handlers OR you can create an `internal` directory in root directory which will contain all the business logic i.e the handlers.